robotino_bringup/robots/uibk-robotino2/urdf/robotino.urdf.xacro

```diff
@@ -20,11 +30,11 @@
     </plugin>
   </gazebo>
 
-  <hokuyo name="hokuyo" parent="base_link">
+  <hokuyo name="hokuyo" parent="base_linkz">
     <origin xyz="${hokuyo_x} ${hokuyo_y} ${hokuyo_z}" rpy="${hokuyo_roll} ${hokuyo_pitch} ${hokuyo_yaw}"/>
   </hokuyo>
 
-  <xacro:arm name="arm_" parent="base_link">
+  <xacro:arm name="arm_" parent="base_linkz">
```

Comment:
something is missed:

`Failed to build tree: parent link [base_linkz] of joint [arm_base_joint] not found. This is not valid according to the URDF spec. Every link you refer to from a joint needs to be explicitly defined in the robot description. To fix this problem you can either remove this joint [arm_base_joint] from your urdf file, or add "<link name="base_linkz" />" to your urdf file.`

see second paragraph https://github.com/squirrel-project/squirrel_robotino/pull/151#issuecomment-246616742